packages/suite/src/components/firmware/Initial.tsx

```diff
@@ -78,26 +86,60 @@ const Body = () => {
 
     const { firmwareRelease } = device;
 
+    // Create custom object containing changelogs for easier manipulation in render() method.
+    // Default changelog format is just a long string where individual changes are separated by "*" symbol.
+    const logsCustomObject: any = {};
+
+    if (firmwareRelease.changelog?.length > 0) {
+        firmwareRelease.changelog.forEach((log: any) => {
+            // get array of individual changes for a given version
+            const logsArr = log.changelog.trim().split(/\*/g);
+
+            // The first element of logsArr is an empty array, so get rid of it (but still make sure it's really empty).
+            if (!logsArr[0]) {
+                logsArr.shift();
+            }
+
+            // Get firmware version, convert to string and use it as a key in custom object
+            const versionString = log.version.join('.'); // e.g. [1,9,8] => "1.9.8"
+
+            logsCustomObject[versionString] = {}; // Object initialization
+            logsCustomObject[versionString].changelogs = logsArr;
+            logsCustomObject[versionString].url = log.url;
+            logsCustomObject[versionString].notes = log.notes;
+        });
+    }
+
     return (
         <BodyWrapper>
-            <H2 isGreen>v{firmwareRelease.release.version.join('.')} has released!</H2>
+            <H2 isGreen>v{firmwareRelease.release.version.join('.')} has been released!</H2>
```

Comment:
this message should be available in Translation component (added to messages.ts) but i guess there are plenty of hardcoded texts in components and i will create another issue for that